test/spec/modules/liveIntentIdSystem_spec.js

```diff
@@ -83,17 +95,6 @@ describe('LiveIntentId', function() {
     }, 200);
   });
 
-  it('should fire an event when getId and a hash is provided', function(done) {
-    liveIntentIdSubmodule.getId({ params: {
-      ...defaultConfigParams,
-      emailHash: '58131bc547fb87af94cebdaf3102321f'
-    }});
-    setTimeout(() => {
-      expect(server.requests[0].url).to.match(/https:\/\/rp.liadm.com\/j\?.*e=58131bc547fb87af94cebdaf3102321f.+/)
-      done();
-    }, 200);
-  });
-
```

Comment:
That test probably should stay 😄

Reply:
🤦‍♀️  it was long time, and forgot that I updated an existing test case
thanks!